optimize/self-managed/optimize-deployment/configuration/system-configuration.md

```diff
@@ -256,23 +254,6 @@ Settings influencing the process digest feature.
-| webhookAlerting.webhooks.$\{webhookName}.url              |               | The URL of the webhook.                                                                                                                                                                                                                             |
-| webhookAlerting.webhooks.$\{webhookName}.headers          |               | A map of the headers of the request to be sent to the webhook.                                                                                                                                                                                      |
-| webhookAlerting.webhooks.$\{webhookName}.httpMethod       |               | The HTTP Method of the request to be sent to the webhook.                                                                                                                                                                                           |
-| webhookAlerting.webhooks.$\{webhookName}.defaultPayload   |               | The payload of the request to be sent to the webhook. This should include placeholder keys that allow you to define dynamic content. See [Alert Webhook Payload Placeholders](../webhooks#alert-webhook-payload-placeholders) for available values. |
-| webhookAlerting.webhooks.$\{webhookName}.proxy.enabled    |               | Whether an HTTP proxy should be used for requests to the webhook URL.                                                                                                                                                                               |
-| webhookAlerting.webhooks.$\{webhookName}.proxy.host       |               | The proxy host to use, must be set if webhookAlerting.webhooks.$\{webhookName}.proxy.enabled = true.                                                                                                                                                |
-| webhookAlerting.webhooks.$\{webhookName}.proxy.port       |               | The proxy port to use, must be set if webhookAlerting.webhooks.$\{webhookName}.proxy.enabled = true.                                                                                                                                                |
-| webhookAlerting.webhooks.$\{webhookName}.proxy.sslEnabled |               | Whether this proxy is using a secured connection (HTTPS). Must be set if webhookAlerting.webhooks.$\{webhookName}.proxy.enabled = true.                                                                                                             |
-
 ### History cleanup settings
 
 Settings for automatic cleanup of historic process/decision instances based on their end time.
```

Comment:
Also on this page, we can delete the following lines:

- plugin.directory
- plugin.variableImport.basePackages
- plugin.authenticationExtractor.basePackages
- plugin.engineRestFilter.basePackages
- plugin.decisionInputImport.basePackages
- plugin.decisionOutputImport.basePackages
- plugin.elasticsearchCustomHeader.basePackages
- serialization.engineDateFormat
- historyCleanup.ingestedEventCleanup.enabled
- historyCleanup.decisionDataCleanup .perDecisionDefinitionConfig.${key}.ttl
- historyCleanup.decisionDataCleanup.perDecisionDefinitionConfig
- historyCleanup.decisionDataCleanup.enabled